autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -11,7 +11,7 @@
 from ..base import BaseTask, compile_scheduler_options
 from ..base.base_task import schedulers
 from ...utils import verbosity2loglevel
-from ...utils.tabular.features.auto_ml_feature_generator import AutoMLFeatureGenerator
+from autogluon.utils.tabular.features.generators import AutoMLPipelineFeatureGenerator
```

Comment:
Should we make sure that we will stick to relative importing? That is `from ...utils.tabular.features.generators import AutoMLPipelineFeatureGenerator`.

Reply:
Yes, this is an issue with my IDE where when I rename a module or move it, it can sometimes switch from relative to absolute import. Please just highlight this whenever it occurs and I can fix.

Reply:
Fixed, I don't see any other absolute imports after checking across the tabular package, although there are absolute imports elsewhere (not in tabular/this PR) that I will create a separate PR to try to resolve.